.env

```diff
@@ -1,7 +1,8 @@
+REACT_APP_API_HOST=https://addons.mozilla.org
 # This is the placeholder used in the `index.html` file to inject the
 # authentication token on runtime.
 REACT_APP_AUTH_TOKEN_PLACEHOLDER=__AUTH_TOKEN__
-
-REACT_APP_API_HOST=https://addons.mozilla.org
-REACT_APP_FXA_CONFIG=amo
 REACT_APP_AUTHENTICATION_COOKIE=frontend_auth_token
+# This is the language that will be sent with all API requests.
+REACT_APP_DEFAULT_API_LANG=en-US
+REACT_APP_FXA_CONFIG=amo
```

Comment:
Any reason to change the order of the variables?

Reply:
I put them in alphabetical order. Without a specific reason for an order, I feel like that's a good practice.